meta-balena-common/recipes-support/balena-info/balena-info.bb

```diff
@@ -1,12 +1,12 @@
-DESCRIPTION = "Resin info"
+DESCRIPTION = "Balena info ascii art"
```

Comment:
Why `ascii art` ?

Reply:
https://github.com/balena-os/meta-balena/blob/51d2f658e973a999f9f346913157b7b4a6979900/meta-balena-common/recipes-support/resin-info/resin-info/resin-info#L1-L12

Reply:
In development images where the console is available, this ascii art is displayed to direct users to the dashboard.